pyracf/common/security_admin.py

```diff
@@ -440,6 +441,10 @@ def __format_data_set_generic_profile_data(
             i += 1
         elif "NO INSTALLATION DATA" in messages[i]:
             profile[current_segment]["installationData"] = None
+        elif "UNLOCKED" in messages[i]:
+            profile[current_segment]["locked"] = False
+        elif "LOCKED" in messages[i]:
+            profile[current_segment]["locked"] = True
```

Comment:
Is there a test to validate this?

Reply:
Nevermind. I see it. This is for APPC session.

Reply:
Yeah, the actual code for the test did not change, but the constants did since previously the only trait being tested was not properly read in.